packages/office-ui-fabric-react/src/components/ThemeGenerator/ThemeGenerator.ts

```diff
@@ -179,4 +178,29 @@ export class ThemeGenerator {
       }
     }
   }
+
+  /* Makes the rest of the code that's used for the load theme blob in the theme designers' codepens.
```

Comment:
Super nitpicky thing
```suggestion
  /* Makes the rest of the code that's used for the load theme blob in the theme designer's codepens.
```

Reply:
There are two theme designers hence the plural - I'll make that more clear.

Reply:
Ahhhh right :)